table/arrow_utils.go

```diff
@@ -417,6 +418,15 @@ func ArrowSchemaToIceberg(sc *arrow.Schema, downcastNsTimestamp bool, nameMappin
 	}
 }
 
+func ArrowSchemaToIcebergWithoutIDs(sc *arrow.Schema, downcastNsTimestamp bool) (*iceberg.Schema, error) {
```

Comment:
I think this name is confusing since we add fresh-IDs?

Reply:
What about:
```suggestion
func ArrowSchemaToIcebergWithFreshIDs(sc *arrow.Schema, downcastNsTimestamp bool) (*iceberg.Schema, error) {
```
Open for other suggestions

Reply:
What about moving this to testing?

Reply:
talked with Fokko separately and we decided we'll keep this in for now (albeit renamed). and remove/rework it in the future if necessary.